db/migrate/20171103140150_create_preneed_submissions.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePreneedSubmissions < ActiveRecord::Migration
+  def change
+    create_table :preneed_submissions do |t|
+      t.string :tracking_number, null: false
+      t.string :application_uuid, null: true
+      t.string :return_description, null: false
+      t.integer :return_code, null: true
```

Comment:
This is `null: false` in the schema below. Should they be matching?

Reply:
An application submission can fail, in which case the application_uuid is may be blank. The return_description should always be present, but the return_code may or may not be present there are issues with the XSD that do not match there DB constraints. In some cases, the EOAS app returns errors in others, their Oracle DB returns errors. This will be more flushed out in the upcoming PR handling application submission errors...

Reply:
There should always be a tracking number...

Reply:
I was specifically referencing that `return_code` is `null: true` in the migration but `null: false` in `db/schema.rb`. Other than that it looks good and makes sense.

Reply:
Good catch!